<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintPreviewBase</class>
 <widget class="QDialog" name="PrintPreviewBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topHLayout">
     <item>
      <widget class="QScrollArea" name="previewArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>744</width>
          <height>663</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settingsBarLayout">
       <item>
        <widget class="QWidget" name="optionWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QGroupBox" name="previewSettings">
            <property name="title">
             <string>Preview Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="PrintLanguageCombo" name="printLanguageCombo"/>
             </item>
             <item>
              <widget class="QComboBox" name="colorOutputMode">
               <item>
                <property name="text">
                 <string>Color</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Grayscale</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mirrorH">
               <property name="text">
                <string>Mirror Page(s) Horizontally</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mirrorV">
               <property name="text">
                <string>Mirror Page(s) Vertically</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="clipToMargins">
               <property name="text">
                <string>Clip to Printer Margins</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="convertSpots">
               <property name="text">
                <string>Convert Spot Colors</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableGCR">
               <property name="text">
                <string>&amp;Under Color Removal</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="displaySettings">
            <property name="title">
             <string>Display Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="antiAliasing">
               <property name="text">
                <string>Enable &amp;Antialiasing</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="showTransparency">
               <property name="text">
                <string>Display Trans&amp;parency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableCMYK">
               <property name="text">
                <string>&amp;Display CMYK</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="inkTable">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="displayInkCoverage">
               <property name="text">
                <string>Display Ink Coverage</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="coverThresholdLabel">
                 <property name="text">
                  <string>Threshold:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ScrSpinBox" name="coverThresholdValue">
                 <property name="decimals">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <double>400.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>10.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>300.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomHLayout">
     <item>
      <widget class="QLabel" name="scaleLabel">
       <property name="text">
        <string>Scaling:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scaleBox">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Resize the scale of the page</string>
       </property>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>150%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fit to Width</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fit to Height</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fit to Page</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PageSelector" name="pageSelector" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>Print...</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>PageSelector</class>
   <extends>QWidget</extends>
   <header>ui/pageselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PrintLanguageCombo</class>
   <extends>QComboBox</extends>
   <header>ui/printlanguagecombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
